### PR TITLE
docs: Explain how to report a security issue.

### PR DESCRIPTION
**Pull request by MaxJPRey**
## Description
As title says.
This will be used for all the other repos of our organization.

## Issue linked
**Please mention the issue number or describe the problem this pull request addresses.**

## Checklist
- [ ] I have tested my changes locally.
- [ ] I have added necessary documentation or updated existing documentation.
- [ ] I have followed the coding style guidelines of this project.
- [ ] I have added appropriate unit tests.
- [ ] I have reviewed my changes before submitting this pull request.
- [ ] I have linked the issue or issues that are solved to the PR if any.
- [ ] I have assigned this PR to myself.
- [ ] I have added the minimum version decorator to any new backend method implemented.
- [ ] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -36,5 +36,14 @@ SOFTWARE.
 
 ## Reporting a Vulnerability
 
+**Please do not report any security vulnerabilities through GitHub issues.**
```
Comment (RobPasMue) [inline SECURITY.md:39]: Why not use GH's admonition system? Looks much better that way... See for ref: https://blog.solichain.com/how-to-create-admonitions-in-github-markdown-ab580debcfdc

Comment (MaxJPRey) [reply SECURITY.md:39]: Excellent idea.

Comment (MaxJPRey) [inline SECURITY.md:39]:
```suggestion
[!CAUTION]
Please do not report any security vulnerabilities through GitHub issues.
```

File: SECURITY.md [hunk 2]
```diff
@@ -36,5 +36,15 @@ SOFTWARE.
 
 ## Reporting a Vulnerability
 
+[!CAUTION]
+Please do not report any security vulnerabilities through GitHub issues.
```
Comment (MaxJPRey) [inline SECURITY.md:40]:
```suggestion
> [!CAUTION]
> Please do not report any security vulnerabilities through GitHub issues.
```

**Review by RobPasMue — APPROVED** [review]
Thanks for the enhancement @MaxJPRey !

**Pull request merged**